src/services/RegisterService.js

```diff
@@ -0,0 +1,18 @@
+import http from './HttpService';
+
+const registration = {
+    register: function (name, email, password, chooseFile) {
+        http.post('/auth/local/register', {
```

Comment:
- Use async await instead of promises here
- The registration flow should be:
1. Register the user (the part that you're already doing)
2. Create a new company on the api ([this endpoint](https://uteam-api-7nngy.ondigitalocean.app/documentation/v1.0.0#/Company/post_companies)) - use a default company name e.g. `${name}'s Company`
3. Upload the profile photo ([this api](https://uteam-api-7nngy.ondigitalocean.app/documentation/v1.0.0#/Upload%20-%20file/post_upload)) ([tutorial](https://docs.strapi.io/developer-docs/latest/plugins/upload.html#upload-files))
4. Create a new profile on the api  ([this endpoint](https://uteam-api-7nngy.ondigitalocean.app/documentation/v1.0.0#/Profile/post_profiles)) - the profile name is the same as user name, the `profilePhoto` is the file you uploaded, and `user` is the user id you got from the `/auth/local/register` call

This was described in https://ablaze-chipmunk-ca4.notion.site/Registration-5e52e85428d041bd95f8d2e521806710